lib/awful/keygrabber.lua

```diff
@@ -508,7 +508,7 @@ end
 -- @tparam awful.key key The key.
 -- @tparam string description.group The keybinding group
 
-function keygrabber:add_keybinding(key, _keycode, _callback, _description)
+function keygrabber:add_keybinding(key, _keycode, _callback, _description) --luacheck: ignore 214
```

Comment:
What's the reason for ignoring these? There'd be no shadowing as far as I can tell.